### ISSUE TITLE
Gurobi interface variable declaration throws error with optlang 1.4.1

### ISSUE THREAD

**Issue by psalvy** [biosustain/optlang#158]
Hi dear optlang team,

I was trying out Gurobi 8.0 with COBRApy, using the optlang interface, and I think something changed in the API that made the gurobi_interface crash at variable initialization. 

You can reproduce the error this way:

```python
from optlang.gurobi_interface import Model, Variable, Constraint, Objective

x1 = Variable('x1', lb=0)
```
```
/usr/local/lib/python3.6/site-packages/optlang/gurobi_interface.py in name(self, value)
    169     @interface.Variable.name.setter
    170     def name(self, value):
--> 171         internal_var = self._internal_variable
    172         super(Variable, Variable).name.fset(self, value)
    173         if getattr(self, 'problem', None) is not None:

/usr/local/lib/python3.6/site-packages/optlang/gurobi_interface.py in _internal_variable(self)
     96     @property
     97     def _internal_variable(self):
---> 98         if self.problem:
     99 
    100             internal_variable = self.problem.problem.getVarByName(self._original_name)

AttributeError: 'Variable' object has no attribute 'problem'
```

my config is the following:
```
optlang==1.4.1
sympy==1.1.1
gurobi==8.0.0
python==3.6.5
```

I am not familiar with the inner workings of optlang, so i admit I am a bit lost as to what might cause this ...
Cheers,
Pierre

EDIT: Added python version
EDIT2: It would seem that I could reproduce the error with Gurobi 7.5 and 7.0. However, *reverting to optlang 1.4.0 fixed  the issue*

**Comment by KristianJensen** [comment]
This is indeed a bug in optlang 1.4.1. Thanks for reporting!

**Comment by KristianJensen** [comment]
This is fixed in 1.4.2
